cli/config/init.go

```diff
@@ -62,7 +62,14 @@ func runInitCommand(cmd *cobra.Command, args []string) {
 		os.Exit(errorcodes.ErrGeneric)
 	}
 
-	msg := "Config file written to: " + configFile
+	var absPath, err = filepath.Abs(destDir)
+	if err != nil {
+		feedback.Errorf("Cannot find absolute path: %v", err)
+		os.Exit(errorcodes.ErrGeneric)
+	}
+	var configFileAbsPath = filepath.Join(absPath, defaultFileName)
```

Comment:
```suggestion
	absPath, err := filepath.Abs(destDir)
	if err != nil {
		feedback.Errorf("Cannot find absolute path: %v", err)
		os.Exit(errorcodes.ErrGeneric)
	}
	configFileAbsPath := filepath.Join(absPath, defaultFileName)
```

Reply:
After applying the suggestion, 
- let's move your code up in the function (to line 52, after the `if destDir == "" {` closing)
- replace `destDir` with your `absPath` in the function
- replace `configFile` with your `configFileAbsPath` in the function

This way it should be more clean :+1: